capture/noworkflow/now/models/graphs/definition_diff_graph.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2017 Universidade Federal Fluminense (UFF)
+# Copyright (c) 2017 Polytechnic Institute of New York University.
+# This file is part of noWorkflow.
+# Please, consult the license terms in the LICENSE file.
+"""Definition Diff Graph Module"""
+from __future__ import (absolute_import, print_function,
+                        division, unicode_literals)
+
+import weakref
+
+from copy import copy
+from collections import defaultdict
+from functools import cmp_to_key
+
+from apted import meta_chained_config, Config, APTED
+from future.utils import viewitems
+
+from .trial_graph import Node
+from .structures import Graph, prepare_cache
+from .diff_graph import CONFIG, children_cmp, create_mapping, merge, merge_edges
+
+
+def create_diff(trial_graph1, trial_graph2):
```

Comment:
This function looks the same as the one in [diff_graph.py](https://github.com/gems-uff/noworkflow/blob/e5cf78953954a471dd6ea3ad4468017b6d3a9363/capture/noworkflow/now/models/graphs/diff_graph.py#L151). Is there a difference?

If there is a difference, try to specialize the existing function with optional parameters. Otherwise, just use the existing one.

Avoid copying and pasting the code because it makes it harder do maintain both versions in the future.